x/dsmr/typed_client.go

```diff
@@ -137,23 +134,14 @@ func (getChunkMarshaler[T]) UnmarshalResponse(bytes []byte) (Chunk[T], error) {
 	if err := proto.Unmarshal(bytes, &response); err != nil {
 		return Chunk[T]{}, err
 	}
-
-	chunk := Chunk[T]{}
-	if err := codec.LinearCodec.UnmarshalFrom(
-		&wrappers.Packer{Bytes: response.Chunk},
-		&chunk,
-	); err != nil {
-		return Chunk[T]{}, err
-	}
-
-	return chunk, chunk.init()
+	return ParseChunk[T](response.Chunk)
 }
```

Comment:
This is almost an unrelated change... This could have previously called to `ParseChunk` as well... But with removal of the `codec.LinearCodec`, I felt like this change made sense here.